### DISCUSSION TITLE
delta pager not working

### DISCUSSION THREAD

**Discussion by prabanjanraja** [jesseduffield/lazygit#3241] [Q&A] [answered]
I have configured delta for syntax highlighting and it works fine with the gitdiff command but even after updating the config.yml file it does not gets reflected in lazygit's diff windows
This is the configuration I used in my config.yml file the other customCommands in the file are working fine for me 
```
git:
  paging:
    colorArg: always
    useConfig: true
```
![image](https://github.com/jesseduffield/lazygit/assets/56388091/48385fa0-60a7-4abb-91f6-4c11e7203789)
![image](https://github.com/jesseduffield/lazygit/assets/56388091/47ee0a77-ff75-4f3e-82e0-e1f2e4b8cdc8)

**Comment by stefanhaller** [comment, marked as answer]
Custom pagers are not supported on Windows yet, sorry.

**Reply by prabanjanraja** [reply]
sorry should have read the docs more carefully
thanks @stefanhaller
